### PR TITLE
polish chapter 06

### PR DESCRIPTION
**Pull request by putcn**

### CODE REVIEW

File: 06.understand_sentiment/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Sentiment Analysis
 
-The source codes of this section can be located at [book/understand_sentiment](https://github.com/PaddlePaddle/book/tree/develop/06.understand_sentiment). First-time users may refer to PaddlePaddle for [Installation guide](https://github.com/PaddlePaddle/book/blob/develop/README.md#running-the-book).
+The source codes of this section locates at [book/understand_sentiment](https://github.com/PaddlePaddle/book/tree/develop/06.understand_sentiment). First-time users may refer to PaddlePaddle for [Installation guide](https://github.com/PaddlePaddle/book/blob/develop/README.md#running-the-book).
```
Comment (abhinavarora) [inline 06.understand_sentiment/README.md]: It should `is located at`

File: 06.understand_sentiment/README.md [hunk 2]
```diff
@@ -28,15 +28,15 @@ The model we used in this chapter uses **Convolutional Neural Networks** (**CNNs
 
 ### Revisit to the Convolutional Neural Networks for Texts (CNN)
 
-The convolutional neural network for texts is introduced in chapter [recommender_system](https://github.com/PaddlePaddle/book/tree/develop/05.recommender_system), here we make a brief overview.
+The convolutional neural network for texts is introduced in chapter [recommender_system](https://github.com/PaddlePaddle/book/tree/develop/05.recommender_system), let's have a brief overview.
```
Comment (abhinavarora) [inline 06.understand_sentiment/README.md]: You can change it to `here is a brief overview`

File: 06.understand_sentiment/README.md [hunk 3]
```diff
@@ -163,9 +163,9 @@ def stacked_lstm_net(input_dim,
                      stacked_num=3):
     """
     A Wrapper for sentiment classification task.
-    This network uses bi-directional recurrent network,
-    consisting three LSTM layers. This configure is referred to
-    the paper as following url, but use fewer layrs.
+    This network uses a bi-directional recurrent network,
+    consisting three LSTM layers. This configure refers to
+    the paper with following url, but use fewer layers.
```
Comment (abhinavarora) [inline 06.understand_sentiment/README.md]: Should be `consisting of three LSTM layers`.

File: 06.understand_sentiment/README.md [hunk 4]
```diff
@@ -163,9 +163,9 @@ def stacked_lstm_net(input_dim,
                      stacked_num=3):
     """
     A Wrapper for sentiment classification task.
-    This network uses bi-directional recurrent network,
-    consisting three LSTM layers. This configure is referred to
-    the paper as following url, but use fewer layrs.
+    This network uses a bi-directional recurrent network,
+    consisting three LSTM layers. This configure refers to
```
Comment (abhinavarora) [inline 06.understand_sentiment/README.md]: `This configuration is motivated from the following paper, but uses few layers`.
Also make the paper a hyperlink and remove the link from the text.

**Review by wangkuiyi — APPROVED** [review]

**Pull request merged**